include/arch/arm/arch.h

```diff
@@ -43,9 +43,11 @@ extern "C" {
 #include <arch/arm/cortex_m/nmi.h>
 #endif
 #if defined(CONFIG_MPU_STACK_GUARD)
-#define STACK_ALIGN  32
+#define STACK_ALIGN		32
+#define STACK_GUARD_SIZE	4
```

Comment:
Hi,

Why do we set STACK_GUARD_SIZE to 4 when stack guard is enabled?
Set it to 32 might be more reasonable since the minimum size of MPU region should be 32.
And set STACK_GUARD_SIZE to 4 will also cause problem since in
configure_mpu_stack_guard(), we will subtract STACK_ALIGN from stack start address.